chore(KFLUXVNGD-43): add tekton-results

avi-biton opened the pull request:
- add patch for `bitnami` permissions on `tekton-results`
- add `tekton-results`
- removing `ui` component because of errors in argo

results of argocd deployment can be seen in https://argocd.konflux-ci.net/applications?showFavorites=false&proj=&sync=&autoSync=&health=&namespace=&cluster=&labels=&page=0 (currently pointing to my repo)

yftacherzog commented:
Are the changes in the UI and trust-manager related to this change? (are we actually making any changes related to trust-manager on this PR?)

avi-biton commented:
You are right, I was just reorganizing the kustomization files.
I updated the description, and removed the `trust-manager` notes from it

As for the UI, I noticed that the Argo CD was failing to to the reference in the kustomization files so I had to remove it to make it work.
